### PR TITLE
Fix node norm category handling

### PR DESCRIPTION
**Pull request by patrickkwang**

### CODE REVIEW

File: tests/query_planner/test_query_planner.py [hunk 1]
```diff
@@ -324,3 +324,39 @@ async def testable_generate_plans():
...
+async def test_bad_norm():
+    """
+    Test that the pinned node "XXX:123" that the normalizer does not know
+    is still handled correctly based on the provided category.
+    """
+
+    qg = {
+        "nodes": {
+            "n0": {
+                "id": "XXX:123",
```
Comment (alongreyber) [inline tests/query_planner/test_query_planner.py:345]: Could you use the new query graph string utility for consistency?

File: tests/query_planner/test_query_planner.py [hunk 2]
```diff
@@ -324,3 +324,39 @@ async def testable_generate_plans():
         testable_generate_plans,
         "generate plan for a typical query graph (50k KPs)",
     )
+
+
+@pytest.mark.asyncio
```
Comment (alongreyber) [inline tests/query_planner/test_query_planner.py:329]: I am trying to keep the tests marked with @pytest.longrun at the bottom of this file. Can you move this test above those?

File: strider/normalizer.py [hunk 3]
```diff
@@ -13,6 +16,12 @@ async def get_types(self, curies):
                 params=dict(curie=curies)
             )
         types = []
+        if response.status_code >= 300:
+            LOGGER.warning(
+                f"Received {response.status_code} response from normalizer: "
+                + response.text
+            )
+            return types
         for c in curies:
```
Comment (alongreyber) [inline strider/normalizer.py:25]: Maybe we should add a warning to the log if we don't get any results back from normalizer. If we do, we should also add a check to the test to make sure the log message exists.

**Review by alongreyber — CHANGES_REQUESTED** [review]

File: tests/query_planner/test_query_planner.py (hunk 1 above)
Comment (patrickkwang) [reply tests/query_planner/test_query_planner.py:345]: I don't think so... The query graph string function appears to take only a single property per node, we need two here. We could probably improve it.

File: strider/normalizer.py (hunk 3 above)
Comment (patrickkwang) [reply strider/normalizer.py:25]: You mean also warn if we get an empty or null 200 response? Yes, that's a good idea.

File: tests/query_planner/test_query_planner.py (hunk 1 above)
Comment (alongreyber) [reply tests/query_planner/test_query_planner.py]: Ah good point. I didn't notice that.

**Review by alongreyber — CHANGES_REQUESTED** [review]
If you rebase this branch onto master and then run the tests, `test_invalid_two_disconnected_components` fails because of an invalid log message. I suspect that this is because MONDO:0005737 is not available in the normalizer. Could you add a fix?

**Comment by patrickkwang** [comment]
Good catch, @alongreyber. If we want, we can enforce that PRs should be up-to-date with main. It leads to more force-pushing than necessary, though.

**Review by alongreyber — APPROVED** [review]

**Pull request merged**